docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG VERSION=0.5.0
+
+FROM golang:1.18-alpine as builder
+
+ARG VERSION
+ARG TARGETOS
+ARG TARGETARCH
+
+
+WORKDIR /workspace
+
+COPY ./ .
+
+RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o pulsarctl -ldflags "-X github.com/streamnative/pulsarctl/pkg/pulsar.ReleaseVersion=Pulsarctl-Go-${VERSION}" .
+
+FROM alpine:3.16.0
+
+RUN apk add --no-cache \
```

Comment:
Could we remove this version?

Reply:
Basically it's a good practice to pin a version of the base image, in case the "latest" image become incompatible overtime, which also makes it reproducible overtime.

Reply:
Good catch, usually we only need to lock the docker image.